cloud/ec2.go

```diff
@@ -1257,22 +1240,20 @@ func (m *ec2Manager) StopInstance(ctx context.Context, h *host.Host, user string
 		"distro":        h.Distro.Id,
 	})
 
-	return errors.Wrap(h.SetStopped(user), "failed to mark instance as stopped in db")
+	return errors.Wrap(h.SetStopped(user), "marking DB host as stopped")
 }
 
 // StartInstance starts a stopped EC2 instance.
 func (m *ec2Manager) StartInstance(ctx context.Context, h *host.Host, user string) error {
-	// Check that target instance is stopped
 	if h.Status != evergreen.HostStopped {
-		return errors.Errorf("cannot start '%s' - host is not stopped", h.Id)
+		return errors.Errorf("cannot start '%s' - host status is '%s'", h.Id, h.Status)
```

Comment:
would it be a benefit to allow HostStarting through here as well?

Reply:
Allowing `HostStarting` would produce some ambiguity in the host lifecycle with not much benefit, since users can't start a host that's in the `HostStarting` state anyways. `HostStarting` is not analogous to `HostStopping`, and specifically refers to the initial host start, not subsequent reboots. `HostStopping` only exists as a convenience warning to the user to indicate that stopping is happening and they should probably stop whatever work they're doing on the host. In contrast, rebooting from stopped to running doesn't require the user to act on it since they won't be able to use the host until it's running anyways (and they get their IP address).

Reply:
I get it now. Thank you for explaining.
As a user I appreciate that something changes on the UI when I click the stop button and the visibility it gives into the host state. I never realized I wasn't getting it for starting.

Reply:
The difference is mostly in whether the user might be interrupted while doing something. For starting a stopped host, the user can't be interrupted while using the host since they don't have access to it anyways, so there's not as much benefit to propagating the restarting state except to remind the user that they pressed the start button. The "stopping" state is possibly more important since the user might still be actively using the host.